v2/svid/jwtsvid/svid.go

```diff
@@ -0,0 +1,15 @@
+package jwtsvid
+
+import (
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
+)
+
+// SVID represents a JWT-SVID.
+type SVID struct {
+	Token string
```

Comment:
I can't imagine any meaningful methods on a token type since you can't do anything with it until it is parsed and the signature validated.
Also, talked with @evan2645  IRL and this field is going to be unexported... (i'm just removing for now).

I'm also going to add a Marshal() method to the SVID type.